package.json

```diff
@@ -18,6 +18,7 @@
     "@rollup/plugin-commonjs": "14.0.0",
     "@rollup/plugin-json": "4.1.0",
     "@rollup/plugin-node-resolve": "8.4.0",
+    "rollup-plugin-terser": "5.1.3",
```

Comment:
expect terser 4.4.0 as defined below. Did a binary search https://openbase.com/js/rollup-plugin-terser/versions:

5.1.0 -> terser 4.1.0
5.1.2 -> terser 4.1.0
5.1.3 -> terser 4.1.0
5.2.0 -> terser 4.6.2

The closest one is 5.1.3